dragonfly/implementations/daq_run_interface.py

```diff
@@ -26,6 +29,7 @@ def __init__(self,
                  directory_path=None,
                  data_directory_path=None,
                  meta_data_directory_path=None,
+                 #Linux_not_Windows_DAQ = True,
```

Comment:
If this isn't used please delete it and the commented lines below that were based on it.